recipes/ddocent/meta.yaml

```diff
@@ -1,14 +1,14 @@
 package:
   name: ddocent
-  version: '2.2.25'
+  version: '2.3.8'
 
 source:
-  fn: v2.2.25.zip
-  sha256: 134a47e4f5703b91483efd338cae8d3e972d160e6a606e27cb1f5fccc71874e6
-  url: https://github.com/jpuritz/dDocent/archive/v2.2.25.zip
+  fn: v2.3.8.zip
+  sha256: 540abc1d7c0f77bcf20835ef764ce3db6956719083d058cd2939433331215fb3
+  url: https://github.com/jpuritz/dDocent/archive/v2.3.8.zip
 
 build:
-  number: 2
+  number: 1
```

Comment:
please reset the build number to zero, when the package version is being updated.